app/src/main/res/layout/bottom_sheet_options_menu_fragment.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.appcompat.widget.LinearLayoutCompat xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There is some indentation issue  ( I am fully sure)

Reply:
Updated with proper indentation